src/dev-app/dev-app/dev-app-layout.html

```diff
@@ -1,31 +1,36 @@
 <mat-sidenav-container class="demo-container">
   <mat-sidenav #navigation role="navigation">
     <mat-nav-list>
-      <a *ngFor="let navItem of navItems"
-         mat-list-item
-         (click)="navigation.close()"
-         routerLinkActive
-         #routerLinkActiveInstance="routerLinkActive"
-         [attr.tabindex]="routerLinkActiveInstance.isActive ? 0 : -1"
-         [routerLink]="[navItem.route]">
+      <a
```

Comment:
The changes in this file all look unrelated.

Reply:
I think the changes from prettier made it hard to see. I've reverted the formatting to make it clear what has changed